Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: all clean
 
 BOOK1=Harry Potter and the Smart-Ass Stones
+BOOK2=Harry Potter and the chamber of ratios
```

Comment:
Should probably capitalize the title to match others - `Harry Potter and the Chamber of Ratios`